front_end/html/edit_assignment.html

```diff
@@ -247,6 +247,25 @@ <h2>Create assignment</h2>
         </p>
     </div>
 
+    <div class="shadow-box">
+        <p><strong>Restrict assignment access to the following IP addresses: </strong></br>Separate valid addresses by a comma without spaces.</p>
+        <div class="select is-grey">
+            <select id="access_restricted" onchange="showAllowedIPAddresses(this.value)" name="access_restricted" class="edit-select">
```

Comment:
I don't think we need to ask them whether to restrict access. We can just give them the text box where they can specify the IP addresses. If they specify IP addresses, then it is implied that access is restricted. If they leave it blank, then it is not restricted.